src/transformers/models/oneformer/dataset_info_oneformer.py

```diff
@@ -0,0 +1,372 @@
+# coding=utf-8
+# Copyright 2022 SHI Labs and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ADE20K_150_CATEGORIES = [
```

Comment:
This file should normally not be included in the model folder.

We just include id2label and label2id mappings in the configuration of the model (model.config), see an example [here](https://github.com/huggingface/transformers/blob/e135a6c93125db85b291380186b840a6b488a151/src/transformers/models/segformer/convert_segformer_original_to_pytorch.py#L134-L157).

Reply:
Thanks for pointing this out! I am only using it for converting the checkpoints to hf format. I will change this according to the example. I found that `huggingface/label-files` repo does not have a JSON file for the `coco-panoptic` dataset. Do you want me to open PR there for that?

Reply:
@praeclarumjj3 that would be great!

Reply:
Yes that'd be great!

Reply:
Hi @NielsRogge @alaradirik, I have one query regarding the JSON files. These only include the mapping from id to name. However, we also need to check whether an ID belongs to the thing classes. So, I will add a file for each dataset (ADE20K, Cityscapes and COCO-Panoptic) containing all the information. I hope that's fine.

Reply:
@praeclarumjj3 that would be great for testing purposes but it might be difficult to check for valid IDs within the post-processing method as users would need to input the dataset name.

Reply:
Hi @alaradirik, I understand your concern. 

When loading pretrained configs, the metadata will be loaded directly, so the user won't have to input any dataset name and just a model name. For training, they will input the dataset name anyways, so it shouldn't be an issue.